### DISCUSSION TITLE
Red syntax errors all of a sudden

### DISCUSSION THREAD

**Discussion by Davesakraken** [mdx-js/mdx#2521] [Q&A]
I've been working on a support site using mdx and have come to my project to find red text all over the files? This was never a problem before and all text was orange. Any ideas why this is happening? If I remove the top heading it seems to fix some of it but unsure why the headings would be effecting this.

**Clip**
https://github.com/user-attachments/assets/8409cab1-dd61-4c93-abf1-d09585340721

**Image**
![image](https://github.com/user-attachments/assets/47f050c6-cefd-448d-9721-5c0d8205409b)

**Comment by ChristianMurphy** [comment]
Duplicate of https://github.com/mdx-js/mdx-analyzer/issues/462, https://github.com/mdx-js/mdx-analyzer/issues/463, https://github.com/mdx-js/mdx-analyzer/issues/464, and https://github.com/wooorm/markdown-tm-language/issues/10

The issue is a bug in VSCode's YAML parser, see the workarounds here: https://github.com/wooorm/markdown-tm-language/issues/10#issuecomment-2265019049
